ci-operator/step-registry/ipi/conf/vsphere/ipi-conf-vsphere-commands.sh

```diff
@@ -13,6 +13,14 @@ base_domain=$(<"${SHARED_DIR}"/basedomain.txt)
 declare -a vips
 mapfile -t vips < "${SHARED_DIR}/vips.txt"
 
+cat > "${SHARED_DIR}"/.awsconf <<EOF
+[profile default]
+region = us-east-1
```

Comment:
This can cause a lot more problems as it would allow users to invoke the cli without specifying the region making it difficult to debug why i can't see my resources. I would recommend on setting this default region if possible.

Reply:
We should be fine to lose the region line in a generic file. It doesn't do anything for us here since route53 is a global service anyway.